crates/uv-workspace/src/pyproject.rs

```diff
@@ -497,6 +497,36 @@ pub struct ToolUv {
     )]
     pub environments: Option<SupportedEnvironments>,
 
+    /// A list of required platforms, for packages that lack source distributions.
+    ///
+    /// When a package does not have a source distribution, it's availability will be limited to
+    /// the platforms supported by its built distributions (wheels). For example, if a package only
+    /// publishes wheels for Linux, then it won't be installable on macOS or Windows.
+    ///
+    /// By default, uv will require that each package includes at least one wheel for Linux, macOS,
+    /// and Windows. The `required-platforms` setting can be used to make those requirements more
+    /// or less strict. For example, an empty `required-platforms` list avoid enforcing any such
+    /// requirements on binary-only packages.
```

Comment:
We should make clearer that the specific requirement is that for each marker, there exists at least one wheel that can be installed in the marker's environment.